_sass/_partners.scss

```diff
@@ -0,0 +1,91 @@
+@media screen and (min-width: 280px) {
+    .each-partner {
+        width: 100%;
+        background-color: #273647;
+    }
+    .partner-logo{
+      width: 200px;
+      height: 8vw;
+      padding: 1%;
```

Comment:
I'd prefer to stick with consistent units in our CSS, so for `padding` and `margin` let's keep these all as `px` units, rather than percentage values or `vw`.